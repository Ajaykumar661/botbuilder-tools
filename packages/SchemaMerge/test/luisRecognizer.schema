{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "title": "LUIS Recognizer",
    "description": "The root definition for components.",
    "$implements": [
        "IRecognizer"
    ],
    "allOf": [
        {
            "$ref": "component.schema#"
        },
        {
            "properties": {
                "$type": {
                    "const": "Microsoft.LUISRecognizer"
                },
                "application": {
                    "type": "object",
                    "$comment": "We need a consistent way to handle keys.",
                    "properties": {
                        "applicationId": {
                            "type": "string"
                        },
                        "endpoint": {
                            "type": "string"
                        },
                        "endpointKey": {
                            "type": "string"
                        }
                    },
                    "required": [
                        "applicationId",
                        "endpoint",
                        "endpointKey"
                    ]
                }
            }
        }
    ]
}