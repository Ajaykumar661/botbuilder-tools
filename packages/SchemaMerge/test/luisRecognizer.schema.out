{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "title": "LUIS Recognizer",
    "description": "The root definition for components.",
    "$implements": [
        "IRecognizer"
    ],
    "type": "object",
    "required": [
        "type"
    ],
    "properties": {
        "$type": {
            "type": "string",
            "const": "Microsoft.LUISRecognizer"
        },
        "$id": {
            "type": "string"
        },
        "application": {
            "type": "object",
            "$comment": "We need a consistent way to handle keys.",
            "required": [
                "applicationId",
                "endpoint",
                "endpointKey"
            ],
            "properties": {
                "applicationId": {
                    "type": "string"
                },
                "endpoint": {
                    "type": "string"
                },
                "endpointKey": {
                    "type": "string"
                }
            }
        }
    }
}